packages/fast-form-generator-react/src/form/form-item.children.style.ts

```diff
@@ -46,9 +45,36 @@ const styles: ComponentStyles<FormItemChildrenClassNameContract, {}> = {
         display: "block",
         marginTop: toPx(12),
     },
-    formItemChildren_addedChildren: {
+    formItemChildren_existingChildren: {
         ...applyCleanListStyle(),
     },
+    formItemChildren_existingChildrenItem: {
+        position: "relative",
+        height: "48px",
+        paddingLeft: "26px",
+        display: "flex",
+        alignItems: "center",
+        "&::before": {
+            position: "absolute",
+            content: "''",
+            opacity: ".6",
+            pointerEvents: "none",
+            top: "calc(50% - 8px)",
+            width: toPx(16),
+            height: toPx(16),
+            background: lines,
+            left: "0",
+        },
+        "& a": {
+            width: "calc(100% - 36px)",
+            "& span, i": {
```

Comment:
@janechu `li`? I don't think that we are using `i` (nor should we as that's a deprecated element).

Reply:
Deprecated, where are you reading that? There is of course `<i>` and `<em>` which is similar but the text used here is not for emphasis, it's italicized to set it apart from the other text in the `<li>`.

See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/i

Reply:
I wasn’t seeing the ‘i’ which is why I was curious. If we are considering this as a semantic representation for “italicized” text, the spec has changed: https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-i-element

I’d suggest if the existing spec doesn’t align and there is no semantic meaning - just presentational italics - we should just style with a span.

Reply:
For context, the `<span>` is used as the name of the child component, the `<i>` is used if there is content for it, they should not be treated the same or in the "same voice" therefore the use of `<i>` to differentiate.

An example would be a component named "Button" with the inner text being "Hello world". You would then read "Button hello world" in the same voice if both used the `<span>` element.

Reply:
Fantastic, thanks for the context!